packages/message-list/src/vaadin-message-mixin.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2024 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
+import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+
+/**
+ * @polymerMixin
+ * @mixes ControllerMixin
+ * @mixes FocusMixin
+ */
+export const MessageMixin = (superClass) =>
+  class MessageMixinClass extends FocusMixin(ControllerMixin(superClass)) {
```

Comment:
nit: should `ControllerMixin` be used only on `vaadin-message.js` so it doesn't end up in the Lit-migrated component?

Reply:
We generally use `ControllerMixin` in other mixins that use `SlotController` internally like the `LabelMixin`.
Also keeping it here is ok since `ControllerMixin` is a no-op in case the superclass is `LitElement`.